recipe/meta.yaml

```diff
@@ -1,49 +1,63 @@
-{% set version = "0.8.29" %}
+{% set version = "0.8.33" %}
+{% set name = "hdbscan" %}
 
 package:
-  name: hdbscan
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/h/hdbscan/hdbscan-{{ version }}.tar.gz
-  sha256: 7e9d7351610eaadddb281e3149a74e22e329bc0b5325f631031d4b63a6a770ae
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 57fabc5f0e45f48d2407b35c731192abc896376411fe7e4bb836ffa03d38f90d
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv
```

Comment:
Some our recipes use -vv and others -vvv, which is the highest level of verbosity available in pip. I don't know if we have a particular preference on the level of verbosity. I don't think this is critical, but I will revert to -vv.

Reply:
I don't think we have an exact policy about the level of verbosity. I got reviewed to prefer `vv`, maybe to shorten the logs, especially on certain repositories. Since I'm not sure, please choose as you wish.

Reply:
The highest level of verbosity will increase the build time without any advantages